src/applications/edu-benefits/1990/containers/IntroductionPage.jsx

```diff
@@ -3,114 +3,688 @@ import { focusElement } from 'platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import ErrorableRadioButtons from '@department-of-veterans-affairs/formation-react/ErrorableRadioButtons';
+import { educationWizard10203 } from '../../selectors/educationWizard';
+import { connect } from 'react-redux';
+import recordEvent from 'platform/monitoring/record-event';
+import classNames from 'classnames';
+import _ from 'lodash/fp';
 
+const levels = [
+  ['newBenefit'],
+  ['serviceBenefitBasedOn', 'transferredEduBenefits'],
+  ['nationalCallToService', 'sponsorDeceasedDisabledMIA'],
+  ['vetTecBenefit'],
+  ['sponsorTransferredBenefits'],
+  ['applyForScholarship'],
+];
 export class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+    // flattens all the fields in levels into object keys
+    this.state = []
+      .concat(...levels)
+      .reduce((state, field) => Object.assign(state, { [field]: null }), {
+        open: false,
+        educationBenefitSelected:
+          sessionStorage.getItem('educationBenefitSelected') || 'none selected',
+        wizardCompletionStatus:
+          sessionStorage.getItem('EduWizardStatus') || 'not complete',
+      });
+  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  componentDidUpdate() {
+    const {
+      newBenefit,
+      serviceBenefitBasedOn,
+      nationalCallToService,
+      transferredEduBenefits,
+      sponsorDeceasedDisabledMIA,
+      sponsorTransferredBenefits,
+      vetTecBenefit,
+      applyForScholarship,
+      wizardCompletionStatus,
+    } = this.state;
+    const { showSTEMScholarship } = this.props;
+    const form1995 = showSTEMScholarship ? '10203' : '1995';
+    if (wizardCompletionStatus === 'pending') {
+      if (
+        newBenefit === 'yes' &&
+        nationalCallToService === 'no' &&
+        vetTecBenefit === 'no'
+      ) {
+        this.setEduBenefitFormSelected('1990');
+        this.setWizardCompletionStatus('complete');
+      } else if (
```

Comment:
can we remove the use of else and just let the if statements remain on their own line (guard clauses)